### ISSUE TITLE
Alt-t Mystery

### ISSUE THREAD

**Issue by technatica** [leo-editor/leo-editor#866]
I had been using Alt-t to focus-to-tree as per the settings:

`/leo-editor/leo/config/leoSettings.leo#@settings-->Keyboard shortcuts-->@keys EKR bindings-->@shortcuts Gui operations`

After the recent work on menus, I noticed that Alt-t was now bound to the "Settings" menu on the first t in Settings. Go ahead and hold down the Alt key and you will see it underlined.

So imagine my surprise when I visited the @menu setting and discovered that Alt-t is **NOT** bound to Settings at all. @menu Settings has no &. All of the other @menu headings do.

`/leo-editor/leo/config/leoSettings.leo#@settings-->Menus-->@menus-->@menu Settings`

Sooo, being the inquisitive type, I thought I would try changing the underline manually. Changing the @menu in myLeoSettings.leo to @menu Settin&gs does not have the intended effect. The underline remains under the first t in Settings. Changing it in leoSettings.leo does have the intended effect of placing the underline under the g in Settings.

But Alt-t now selects the "Tags" tab in the log pane.

When I remove the nodetags.py plugin, Alt-t does nothing at all, even though focus-to-tree = Alt-t in both myLeoSettings.leo and leosettings.leo.

**Comment by edreamleo** [comment]
On Fri, Apr 20, 2018 at 9:21 PM, Chris George <notifications@github.com>
wrote:

>
> imagine my surprise when I visited the @menu <https://github.com/menu>
> setting and discovered that Alt-t is *NOT* bound to Settings at all. @menu
> <https://github.com/menu> Settings has no &. All of the other @menu
> <https://github.com/menu> headings do.
>
​On Windows, and perhaps other platforms, Alt keys bound to menus generate
no key events at all.  The system handles them.

This probably explains the behaviors you note.

Edward

**Comment by technatica** [comment]
I am on Linux. Alt-t has worked fine to focus-to-tree (which is the default shortcut in leoSettings.leo) up until the menu reworking.

Should Alt highlight the first t in settings if there is no & in Settings in the @menu Settings setting?

**Comment by edreamleo** [comment]
On Wed, Apr 25, 2018 at 8:31 PM, Chris George <notifications@github.com>
wrote:

> I am on Linux. Alt-t has worked fine to focus-to-tree (which is the
> default shortcut in leoSettings.leo) up until the menu reworking.
>
​Hmm.  It continues to work for me on both Windows and Linux.

Edward

**Comment by technatica** [comment]
Leo 5.7.2 devel, build 20180423091843, Mon, Apr 23, 2018 9:18:43 AM
Git repo info: branch = devel, commit = f76ab4119b31
Python 3.6.4, PyQt version 5.10.1
linux
isPython3: True
caching enabled


Hmmm.

I installed Leo fresh from git. I backed up my .leo directory and allowed
Leo to create a new one. I used a brand new, default myLeoSettings.leo. And
the behaviour persists.

Alt-t is bound to the second t in Settings. The @menu Settings in
leoSettings.leo does not have an &. Alt-t is the default binding for
focus-to-tree in leoSettings.leo.

I can change the key to g using @menu Settin&gs in myLeoSettings.leo. Then
Alt-t activates the tag pane. I remove the nodetags plugin and Alt-t does
nothing at all, even though it is defined as a shortcut in both
leoSettings.leo and myLeoSettings.leo.

I have already stopped trying to use Alt-t to change panes. I now use
Ctrl-t to toggle pane which achieves the same thing. I just think it odd
that @menu Settings (no &) is hard coded to Alt-t and that the old
behaviour doesn't work any longer for me.

I don't know what more I can do to test this. As with the "Find" issue, it
seems that no matter what I do the behaviour on my system is broken, while
on your systems it works fine. Ctrl-f or mouseclick Find tab (with no
string selected) still selects the first word of body text and plunks it
into the find dialogue, unless the body lacks text, in which case it opens
the Find pane with <find pattern here> selected and the insertion point
wherever it was before the Ctrl-f. If I select a string before hitting
Ctrl-f, the string is plunked into the find dialogue, just as expected.

Changing to a different python3/pyQt configuration makes no difference.
Changing to python2 makes no difference. I will set-up a VM I guess and
test against Ubuntu or something. I do not have access to a Windows
computer or even a Windows OS, other than an old copy of XP somewhere.

Any ideas gratefully accepted. I find myself reluctant to use "Find" as the
behaviour is just annoying.

Chris
